src/inspirehep-holdingpen-js/holdingpen/holdingpen.services.js

```diff
@@ -83,6 +83,18 @@
               });
             },
 
+            onBestMatchSelected: function(vm, workflowId, match, showPossibleMatches) {
+              var data = JSON.stringify({
+                'match_recid': match 
+              });
+              
+              $http.post('/api/holdingpen/' + workflowId + '/action/resolve', data).then(function() {
+                showPossibleMatches.push(1);
```

Comment:
It is almost always bad to have output parameters, can't we have this on `vm` if it is needed on UI?

Reply:
In fact having those as arrays so that you carry reference and push some dummy element to have UI state, is confusing and too hackish even perhaps for this codebase.